deploy/crds/contrail.juniper.net_managers_crd.yaml

```diff
@@ -1496,12 +1496,60 @@ spec:
                             properties:
                               cassandraInstance:
                                 type: string
+                              cassandraNodesConfiguration:
```

Comment:
In my opinion static configuration shouldn't be a part of manager crds.

Reply:
Should it be put aside of CommonConfiguration and ServiceConfiguration as separate thing in vRouter spec?

Reply:
Yes

Reply:
This is how I see it

Reply:
Actually this ServiceConfiguration is the same struct in Manager and vRouter, it doesn't need to be like this.

Reply:
Ok, I moved static configuration out of manager and as separate struct field in spec